src/SecretsMocker/SecretsMocker/Dockerfile

```diff
@@ -1,20 +1,16 @@
-FROM --platform=$BUILDPLATFORM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
```

Comment:
This was the thing messing you up I believe.

Reply:
Unfortunately, if I remove this, the build fails. See [the errors ](https://twitter.com/timheuer/status/1757137301250232675)